src/components/FilterBar.test.jsx

```diff
@@ -0,0 +1,37 @@
+import { render, screen } from '@testing-library/react';
+import queryString from 'query-string';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import FilterBar from './FilterBar';
+
+const renderWithTheme = (ui) =>
+  render(<ThemeProvider theme={createTheme()}>{ui}</ThemeProvider>);
+
+const queryOptions = {
+  arrayFormat: 'bracket-separator',
+  arrayFormatSeparator: ',',
+};
+
+const params = queryString.parse(
+  { sortBy: 'amount.asc' },
+  {
+    parseNumbers: true,
+    ...queryOptions,
+  }
+);
+
+const setQueryParam = () => {};
+
+describe('HomeSection', () => {
+  test('renders the component', () => {
+    renderWithTheme(<FilterBar queryParams={params} {...{ setQueryParam }} />);
```

Comment:
not testing anything here, but if you write tests checking for `sort` and `amount filter` buttons, then that would imply that `FilterBar` is rendering